docker-compose.yml

```diff
@@ -16,8 +16,10 @@ secrets:
 services:
   gophish:
     build:
-      # VERSION must be specified on the command line:
-      # e.g., --build-arg VERSION=0.0.1
+      # Note build arguments here are for builds using docker-compose (pytest)
+      # The CI build workflow uses docker and passes the value from bump_version
+      args:
+        - VERSION=0.0.5
```

Comment:
Do we want to modify `bump_version.sh` so that this version number is also updated?

Reply:
See 6cc48c2

Reply:
Very nice!  Is this something that we should push up to `skeleton-docker`?

Reply:
Yes.